solr/solr-ref-guide/modules/configuration-guide/pages/replica-placement-plugins.adoc

```diff
@@ -184,6 +192,24 @@ This property defines an additional constraint that collections (keys) must be l
 Nodes are labeled using the `node_type` system property with the value being an arbitrary comma-separated list of labels.
 Correspondingly, the plugin configuration can specify that a particular collection must be placed only on the nodes that match at least one of the (comma-separated) labels defined here.
 
+`spreadAcrossDomains`::
++
+[%autowidth,frame=none]
+|===
+|Optional |Default: `false`
+|===
++
+Spread replicas for a given shard/replica type across spread domains, that are defined by nodes using the system property `spread_domain`
+
+`maxReplicasPerShardInDomain`::
++
+[%autowidth,frame=none]
+|===
+|Optional |Default: `-1`
+|===
++
+Set a maximum number of replicas (for a given shard and replica type) that can be placed in a particular spread domain. Fail the request if this can't be satisfied. This can be used only in combination of `spreadAcrossDomains`.
```

Comment:
```suggestion
Set a maximum number of replicas (for a given shard and replica type) that can be placed in a particular spread domain. Fail the request if this can't be satisfied. This can be used only in combination with `spreadAcrossDomains`.
```